stakingModel/contracts/SimpleDAIStaking.sol

```diff
@@ -144,10 +147,10 @@ contract SimpleDAIStaking is DSMath, Pausable, SchemeGuard {
         if (daiWorth < totalStaked) {
             return (0, 0, 0);
         }
-        uint256 daiGains = daiWorth - totalStaked;
+        uint256 daiGains = daiWorth.sub(totalStaked);
         uint256 cdaiGains = rdiv(daiGains * 1e10, er); //mul by 1e10 to equalize precision otherwise since exchangerate is very big, dividing by it would result in 0.
         uint256 precisionLossCDaiRay = cdaiGains % 1e19; //get right most bits not covered by precision of cdai which is only 8 decimals while RAY is 27
-        if (cdaiGains > 0) {
+        if (cdaiGains > 0) { // otherwise 0
```

Comment:
the comment here is that dividing 0 gives non 0 no?

Reply:
waiting

Reply:
a comment has added